applications/nrf_desktop/src/modules/ble_state.c

```diff
@@ -215,15 +222,16 @@ static void security_changed(struct bt_conn *conn, bt_security_t level)
 
 	struct bt_conn_info info;
 
-	int err = bt_conn_get_info(conn, &info);
-	if (err) {
+	int rc = bt_conn_get_info(conn, &info);
```

Comment:
@mbolivar Please leave it as is for the sake of consistency. Instead rename the new argument to `bt_err`